src/i18n/TextCollection/i18nTextCollector.php

```diff
@@ -472,8 +472,11 @@ private function getModulesAndThemes(): array
         if (!$this->modulesAndThemes) {
             $modules = ModuleLoader::inst()->getManifest()->getModules();
             // load themes as modules
-            $themes = array_diff(scandir(THEMES_PATH), ['..', '.']);
-            if ($themes) {
+            $themes = [];
+            if (file_exists(THEMES_PATH)) {
```

Comment:
```suggestion
            if (is_dir(THEMES_PATH)) {
```

Reply:
`file_exists` will work in most scenarios, but `is_dir` is better here since it checks if the path exists and also is a dir, so a file won't pass whereas incorrectly set `THEMES_PATH` pointing to a file would pass here.

Reply:
Updated